packit/cli/update.py

```diff
@@ -67,6 +67,12 @@
         "If this is not specified, default to origin."
     ),
 )
+@click.option(
+    "--upstream",
+    default=None,
+    help="Git ref that points to the state of the upstream."
```

Comment:
```suggestion
    help="Git ref of the last upstream commit in the current branch from which packit should generate patches (this option implies the repository is source-git)."
```
This should be more clear.

Reply:
I am only not sure about the word `commit`. It can be a tag as well.

Reply:
But the tag points to a commit :D

Reply:
You are right, but I can be misleading..;-)